examples/eon-pet-neb/eon-pet-neb.py

```diff
@@ -0,0 +1,667 @@
+#
+# Here we use the IDPP from ASE to setup the initial path. You can find
+# more information about this method in the corresponding
+# `ASE tutorial <https://ase-lib.org/examples_generated/tutorials/neb_idpp.html>`_
+# or in the original IDPP publication by
+# `S. Smidstru et al. <https://doi.org/10.1063/1.4878664>`_ .
+
+N_INTERMEDIATE_IMGS = 10
+# total includes the endpoints
+TOTAL_IMGS = N_INTERMEDIATE_IMGS + 2
+images = [reactant]
+images += [reactant.copy() for img in range(N_INTERMEDIATE_IMGS)]
+images += [product]
+
+neb = NEB(images)
+neb.interpolate("idpp")
+
+# %%
+# We don't cover subtleties in setting the number of images, typically too many
+# intermediate images may cause kinks but too few will be unable to resolve the
+# tangent to any reasonable quality.
+
+
+# %%
+# For EON, we write the initial path to a file called ``idppPath.dat``.
+#
+
+output_dir = "path"
+os.makedirs(output_dir, exist_ok=True)
+
+output_files = [f"{output_dir}/{num:02d}.con" for num in range(TOTAL_IMGS)]
+
+for outfile, img in zip(output_files, images):
+    ase.io.write(outfile, img)
+
+print(f"Wrote {len(output_files)} IDPP images to '{output_dir}/'.")
+
+summary_file_path = "idppPath.dat"
+
+with open(summary_file_path, "w") as f:
+    for filepath in output_files:
+        abs_path = os.path.abspath(filepath)
+        f.write(f"{abs_path}\n")
+
+print(f"Wrote absolute paths to '{summary_file_path}'.")
+
+# %%
+# ASE
+# ---
+#
+
+
+# define the calculator
+def mk_mta_calc():
+    return MetatomicCalculator(
+        "pet-mad-v1.0.2.pt",
+        device="cpu",
+        non_conservative=False,
+        uncertainty_threshold=0.001,
+    )
+
+
+# set calculators for images
+ipath = [reactant] + [reactant.copy() for img in range(10)] + [product]
+for img in ipath:
+    img.calc = mk_mta_calc()
+
+print(img.calc._model.capabilities().outputs)
+
+neb = NEB(ipath, climb=True, k=5, method="improvedtangent")
+neb.interpolate("idpp")
+
+# plot the initial energy path guess
+energies = [i.get_potential_energy() for i in ipath]
+plt.plot(energies)
+
+# setup the NEB clalculation
+optimizer = LBFGS(neb, trajectory="A2B.traj", logfile="opt.log")
+conv = optimizer.run(fmax=0.01, steps=100)
+
+print("Check if calculation has converged:", conv)
+
+if conv:
+    print(neb)
+
+energies = [i.get_potential_energy() for i in ipath]
+plt.plot(energies)
+plt.show()
+
+# %%
+# In the 100 NEB steps we took, the structure did unfortunately not converge.
+# The metatomic calculator for PET-MAD v1.0.2 provides `LLPR based energy
+# uncertainities <https://atomistic-cookbook.org/examples/pet-mad-uq/pet-mad-uq.html>`_.
+# As we obtain a warning that the uncertainty of the path structure sampled is
+# very high, we stop after 100 steps.
+# The ASE algorithm with LBFGS optimizer does not
+# find good intermediate structures and does not converge
+# at all. Our test showed that the FIRE
+# optimizer works better in this context, but still takes over 500 steps
+# to converge.
```

Comment:
If the FIRE optimizer works better, wouldn't it be more fair for ASE to use it? :sweat_smile: